iroha_data_model/src/lib.rs

```diff
@@ -2351,7 +2363,7 @@ pub mod transaction {
         /// [`Transaction`] payload.
         pub payload: Payload,
         /// [`Transaction`]'s [`Signature`]s.
-        pub signatures: Vec<Signature>,
+        pub signatures: BTreeSet<Signature>,
```

Comment:
Please check if this changes the schema - if so - you need to add api_changes tag and notify client devs in I2::Lib

Reply:
That does i guess, but not scale codec ser/de.

Reply:
Then we still need to notify - so that they understand the schema changes